packages/nextjs/test/config.test.ts

```diff
@@ -300,6 +300,19 @@ describe('webpack config', () => {
     expect(finalWebpackConfig).toEqual(expect.objectContaining(materializedUserWebpackConfig));
   });
 
+  it('allows for the use of `hidden-source-map` as `devtool` value for client-side builds', async () => {
```

Comment:
We could add test that it doesnt modify the config for server builds, but it's not that important, so your call

Reply:
It occurred to me. Couldn't tell if it was overkill. But, hey, why not?